adaptive_shield/README.md

```diff
@@ -0,0 +1,14 @@
+## Overview
+Adaptive Shield's SaaS Security Posture Management solution (SSPM) features proactive, deep, continuous and automated monitoring and management of all business critical SaaS applications. As a SaaS app that integrates with all SaaS apps within minutes, Adaptive Shield enables security teams to gain complete control over their SaaS apps, allowing them to harden their security policies and reduce risks.
+
+With the Adaptive Shield integration, you can track and monitor SaaS posture alerts, such as Configuration Drifts, Integration Failures and Security Checks Degradation, in the events stream in Datadog.
+
+## Setup
+
+1. In Adaptive Shield, go to **"Settings"**
+2. Click on **Alerts** -> **"Add New Alerts Channel"** -> Select **"Datadog"**
+3. Provide alias name
+4. Complete the process by using OAuth 
```

Comment:
What's the flow here? Is the user expected to kick off Authorization from the alerts page within Adaptive Shield? The user can also kick off authorization from the integration tile, so it would be helpful to list that out as an option here too

Reply:
Yes, once a user clicks "Connect Accounts" on the integration tile, they will be redirected to the onboarding_url that you defined as part of your OAuth (this is commonly a login page for your platform).